include/wx/dcsvg.h

```diff
@@ -40,13 +40,26 @@ class WXDLLIMPEXP_CORE wxSVGBitmapHandler
 };
 
 // Predefined standard bitmap handler: creates a file, stores the bitmap in
-// this file and uses the file URI in the generated SVG.
+// this file and uses the file name in the generated SVG.
 class WXDLLIMPEXP_CORE wxSVGBitmapFileHandler : public wxSVGBitmapHandler
 {
 public:
+    wxSVGBitmapFileHandler()
+        : m_path()
+    {
+    }
+
+    explicit wxSVGBitmapFileHandler(const wxString& path)
```

Comment:
Again, not really sure about it, but perhaps we should take `wxFileName` here?

Reply:
I wasn't sure either. I used `wxString` so there is no need for forward declarations. But using `wxFileName` might be more appropriate and future proof. I'll update it.